site/content/blogs/release-v126.en.md

```diff
@@ -36,6 +36,90 @@ Copilot v1.26 brings several new features and improvements:
 
 ## `storage init` for environment addons
 
+Previously, `copilot storage init` only supported storage addons attached to workloads: you need to run 
+`copilot svc deploy` in order to deploy the storage, and the storage is deleted along with the service
+when you run `copilot svc delete`.
+
+Now, you have the option to create environment-level storage addons: the storage is deployed when you run `copilot env deploy`,
+and isn't deleted until you delete the whole environment by running `copilot env delete`.
```

Comment:
```suggestion
and won't be deleted until you delete the environment by running `copilot env delete`.
```

Reply:
removed the "whole". I kept "isn't" though since it was suggested by https://github.com/aws/copilot-cli/pull/4511#discussion_r1109034727 and looks cleaner